_episodes/02-lists.md

```diff
@@ -20,8 +20,7 @@ list[2:9]), in the same way as strings and arrays."
 - "Strings are immutable (i.e., the characters in them cannot be changed)."
 ---
 
-Just as a `for` loop is a way to do operations many times,
-a list is a way to store many values.
+A list is a way to store many values.
```

Comment:
I think we need a better prose here. Maybe even rewrite some of the text below. It would be good to state our goal here: we want to store names of the files to be processed in one place. Maybe we could say that our hypothetical collaborator informed us that he or she has sent us the following files: ...`  What do you think, @ngrafgr and @annefou?

Reply:
Sorry but I am a bit confused. Do you mean you would like to add an example where we create a list of strings?

Reply:
I am confused as well...

Reply:
What I mean is that we can't start talking about lists out of the blue. We need to explain what we want to do, then introduce lists and explain how they enable us accomplish what we need. In episode 1 we already dealt with entities that store "multiple values" - numpy arrays. So, if we simply say "hey, Python has lists" - we can except "raised eyebrows" and questions like: can't/shouldn't we use numpy arrays for all things related to "multiple values"?

Reply:
A gentle ping, @ngrafgr!